app/assets/javascripts/spotlight/crop.es6

```diff
@@ -238,9 +238,11 @@ export default class Crop {
   }
 
   // Get all the form data with the exception of the _method field.
+  // Append the document's authenticity token to the form data.
   getData() {
     var data = new FormData(this.form[0]);
     data.append('_method', null);
+    data.append('authenticity_token', $('meta[name="csrf-token"]').prop('content'));
     return data;
```

Comment:
In the sir-trevor code, we use `ujs.csrfToken()` to grab the CSRF token.

Reply:
Oh, and it looks like we might set the auth token in a request header?

```
      headers: {
        'X-CSRF-Token': ujs.csrfToken() || ''
      },
```

Reply:
👌